src/diffusers/schedulers/scheduling_deis_multistep.py

```diff
@@ -171,7 +171,6 @@ def __init__(
         self.model_outputs = [None] * solver_order
         self.lower_order_nums = 0
 
-    # Copied from diffusers.schedulers.scheduling_dpmsolver_multistep.DPMSolverMultistepScheduler.set_timesteps
```

Comment:
Because it's no longer the same (as we added `use_karras_sigmas`). So, in a separate PR, we can add support for Karras sigmans to this scheduler too. 

Cc: @patrickvonplaten

Reply:
Makes sense!